source/community/reactnative/src/adaptive-card.js

```diff
@@ -166,10 +175,21 @@ export default class AdaptiveCard extends React.Component {
 	}
 
 	getAdaptiveCardContent() {
+		const contentSize = this.props.contentSize
+		const { height } = Dimensions.get('window')
+		var contentHeight = (contentSize && contentSize < height) ? contentSize : height;
+		const scrollEnabled = this.state.screenHeight > contentHeight;
+		const shouldAddHeight = (contentSize < height - 64)
+		var containerStyles = [styles.container]
+		if (shouldAddHeight) {
+			containerStyles.push({ height: contentHeight })
+		}
 		var adaptiveCardContent =
 			(
-				<ContainerWrapper style={styles.container} json={this.state.cardModel}>
-					<ScrollView alwaysBounceVertical={false} style={{ flexGrow: 0 }}>
+				<ContainerWrapper style={containerStyles} json={this.state.cardModel}>
+					<ScrollView
```

Comment:
Scrollable content is not fully visible.

Reply:
Updated scroll view logic